Formula/totp-cli.rb

```diff
@@ -0,0 +1,21 @@
+class TotpCli < Formula
+  desc "Authy/Google Authenticator like TOTP CLI tool written in Go"
+  homepage "https://yitsushi.github.io/totp-cli/"
+  url "https://github.com/yitsushi/totp-cli/archive/v1.2.4.tar.gz"
+  sha256 "0451172dbd85ecddbe5faf14fecd7cefab3e8678ff6b8d6f5cb6cfa8d7defa3b"
+  license "MIT"
+  head "https://github.com/yitsushi/totp-cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    assert_match("totp-cli v", shell_output("#{bin}/totp-cli version"))
+    _, stderr, = Open3.capture3("#{bin}/totp-cli list </dev/null")
+    assert_match("Password", stderr)
```

Comment:
Does this also work?
```suggestion
    assert_match "Password", shell_output("#{bin}/totp-cli list")
```

Reply:
Unfortunately no, however `pipe_output` plus `2>&1` made it :-)